### ISSUE TITLE
Hack-me-if-you-can

### ISSUE THREAD

**Issue by ryanermita** [mozillaph/curriculum#4]
**Activity Name:** Hack-me-if-you-can
*by:* Ryan Ermita, Marion Frigillana and Charlisse Limbitco.

**Overview:**
Participants will learn how the communication works on the internet by playing a traditional Filipino game called [Patintero](https://en.wikipedia.org/wiki/Traditional_games_in_the_Philippines#Patintero).

**Duration:** 
45 minutes

**Leaning Objectives:**
Awareness on how the *secured communication* on the internet works.

**Audience:**
10 y/o and above

**Materials:**
  - Paper
  - Scissors
  - Tape
  - Chalk
  - Markers
  - Pen

**Preparations**
  - Get 5 participants to act as hackers and another 5 to act as the communication team
  - using the chalk, draw the game area, sample photo:
![alt text](https://assets.rappler.com/16EE93FEA3104C39B9B98BB2059EBEC5/img/AD9E26E723E3401B96605BF7E74329B4/patintero-lines-20151008.jpg "Patintero")
  - Ask the communication team to prepare a simple message and write each part of the message in 3 papers.
  - Ask the communication team to have 3 members to work as data packets to deliver the message.
  - Prepare power-ups and let the communication team to randomly pick 3 power-ups.

**Activity Flow**
  - Explain that the game is just like the ([Patintero](https://en.wikipedia.org/wiki/Traditional_games_in_the_Philippines#Patintero) game), Here's the simple video tutorial on how to play [Patintero](https://www.youtube.com/watch?v=YEJAdf5g0U4)
  - Give each team 5 minutes to strategize.
  - The goal of the game is to deliver the message from sender to the recipient by crossing the lines.
  - The 3 members of the communication team will be the data packets who will deliver the message.
  - The goal of the hacker team is to prevent the data packets to deliver the whole message to the recipient by catching the data packets during the game.
- The data packets have 3 power-ups which they can use during the game. This power-ups are randomly picked by the communication team. Sample power-ups (you can create your own)
    - Antivirus (respawn)
    - Firewall (untoucheable wall corner)
    - VPN (move to other box through an outside way)
    - Private Browsing (hacker wont be able to use 1 of his arm)
    - Additional Time (60 seconds)
  - The communication team have only 5 minutes to deliver the message to the recipient or else the hacker team wins.

**Reflection:**
   - How the secured messages works on the internet.
   - Ways to secure communication over the internet
   - How data transfer from the sender and recipient (v/v) in secure way
   - How hackers can intercept data over the internet.

**Comment by tinaverbo** [comment]
For participants who are not familiar with the game Patintero, how about a video instruction?

**Comment by ntayungao** [comment]
Create an animation video where instructions and roles are explained and real life video of a brief overall game.

**Comment by tinaverbo** [comment]
@marquezjohnchristian can we contact the creative peeps who'll be willing to do the animated video for this one?

**Comment by ryanermita** [comment]
while waiting for the creative peeps. I'll add a simple vid tutorial from youtube on how to play Patintero.

**Comment by ryanermita** [comment]
added a link in activity flow on how to play patintero
https://www.youtube.com/watch?v=YEJAdf5g0U4

**Comment by dzeysimarkez** [comment]
@tinaverbo Yup, we can. I already asked for their schedule and availability.

**Comment by ryanermita** [comment]
Thanks @tinaverbo @marquezjohnchristian ! 👍
